File: Nível 1/2342 - Overflow/main.py
maxRepresentacao = int(input())
n1, operador, n2 = map(str, input().split())
n1 = int(n1)
n2 = int(n2)
calc = 0

if operador == "*":
    calc = n1 * n2
else:
    calc = n1 + n2 

if(calc > maxRepresentacao):
    print("OVERFLOW")
else:
    print("OK")   

File: Nível 1/2342 - Overflow/main.lua
maxRepresentacao = tonumber(io.read())
n1, operador, n2 = io.read():match("(%S+)%s+(%S+)%s+(%S+)")
n1 = tonumber(n1)
n2 = tonumber(n2)
calc = 0

if operador == "*" then
    calc = n1 * n2 
else
    calc = n1 + n2
end

if calc > maxRepresentacao then 
    print("OVERFLOW")
else
    print("OK")
end